subprojects/integ-test/integ-test.gradle

```diff
@@ -25,8 +25,8 @@ integTestTasks.all {
     inputs.files(fileTree(userguideOutput))
 
     doFirst {
-        systemProperties['integTest.userGuideInfoDir'] = project(':docs').docbookSrc
-        systemProperties['integTest.userGuideOutputDir'] = userguideOutput.absolutePath
+        fileSystemProperty 'integTest.userGuideInfoDir', project(':docs').docbookSrc
```

Comment:
I think this is an input too, because it contains the `samplesList.xml` needed by the integration tests